js/flightlog.js

```diff
@@ -1145,9 +1170,27 @@ FlightLog.prototype.rcCommandRawToThrottle = function(value) {
     return Math.min(Math.max(((value - this.getSysConfig().minthrottle) / (this.getSysConfig().maxthrottle - this.getSysConfig().minthrottle)) * 100.0, 0.0),100.0);
 };
 
-FlightLog.prototype.rcMotorRawToPct = function(value) {
+FlightLog.prototype.rcMotorRawToPctEffective = function(value) {
+
     // Motor displayed as percentage
     return Math.min(Math.max(((value - this.getSysConfig().motorOutput[0]) / (this.getSysConfig().motorOutput[1] - this.getSysConfig().motorOutput[0])) * 100.0, 0.0),100.0);
+
+};
+
+FlightLog.prototype.rcMotorRawToPctPhysical = function(value) {
+
+    // Motor displayed as percentage
+    let motorPct;
+    if (this.getSysConfig().fast_pwm_protocol < FAST_PROTOCOL.indexOf('DSHOT150')) {
+        // Analog protocol
+        const ANALOG_RANGE = this.getSysConfig().maxthrottle - ANALOG_MIN_VALUE;
```

Comment:
This is inconsistent with the rest - the range should be 2000 - 1000 to indicate the maximum possible range, just like it is for Dshot.

Reply:
Actually I was wrong - this should use `maxthrottle` **and `minthrottle`**, and should also use `minthrottle` to normalise the offset.

Reply:
I was unsure too when I implemented it. The objective of this new motor vision is to show the complete "physical" motor range, from motor stop, to motor at its physical max. In dshot is easy, it has a range from 48 to 2047 for any motor, but in analog I was unsure.

If I remember right, in analog protocols we needed to do a "calibration" that restricts the range that can be pased to the values `minthrottle` to `maxthrottle`, but I was not sure if `minthottle` was the motor stop or was the "analog idle" value.
 
If it is the motor stop, I think you are right and I we need to restrict to it. If it is the "idle" value then I think we need to represent the motor stop too to maintain coherence between both protocols and we need to send lower values too to show the motor stop value, and for this reason I use the 1000 value.

As I say, I was not too sure, it has been so long since I used analog protocols...

Reply:
min_throttle is the lowest sent to any motor while flying, and normally would reflect a stable idle at zero stick.
min command is the value to be sent when the motors should be stopped.
In the old motors display we saw 0% when at minthrottle and 100% when at max throttle, both in graph and numbers.
From the perspective of this PR, it is not quite the same as the DShot situation.  With DShot, zero is always some consistent point below where the motors actually spin.  The relationship between the users idle point and the zero point is reasonably consistent, and has some meaning, especially when dynamic idle is present.
For analog protocols, min_command can be set much lower than the spin-up point.  This is of no particular consequence once flying, and of course dynamic idle cannot be applied.  
By my thinking there is no need, and no benefit, of displaying the offset between min_command and min_throttle in any motors display.  In the past we only ever showed a simple percentage above min_throttle, and we've never had a complaint about that, and I see no virtue in changing this.
For the analog protocols the only useful change could have been to see the actual PWM value that was being sent to the motors, in brackets after the percentage.  Whether it is actually useful to display this, I don't know, it's never been a request and to date I haven't envisaged a use for it.